packages/slate-types/src/sdk/PressContact.ts

```diff
@@ -1,14 +1,19 @@
 export default interface PressContact {
     avatar_url: string | null;
     company: string | null;
+    // TODO: cultures
```

Comment:
Note: in Newsroom Contacts API v2 this property was renamed to `display_locales`.